docs/graphql/manual/index.rst

```diff
@@ -12,12 +12,62 @@ Hasura GraphQL engine documentation
   :depth: 1
   :local:
 
-The Hasura GraphQL engine lets you set up a GraphQL server and event triggers over a Postgres database in minutes.
+The Hasura GraphQL engine connects to your databases & microservices and auto-generates a production-ready GraphQL backend in minutes.
 
+.. list-table::
+   :header-rows: 1
+   :class: toc-table
+
+   * - Basics
+     - Business Logic
+     - Auth
+   * - `Getting Started <getting-started/index>`_
+     - `Actions <actions/index>`_
+     - `Authentication <auth/authentication/index>`_
+   * - `Schema <schema/index>`_
+     - `Remote Schemas <remote-schemas/index>`_
+     - `Authorization <auth/authorization/index>`_
+   * - `Queries <queries/index>`_
+     - `Event Triggers <event-triggers/index>`_
+     - `Access Control Basics <auth/authorization/basics>`_
+   * - `Mutations <mutations/index>`_
+     - `Scheduled Triggers <scheduled-triggers/index>`_
+     - `Roles & Session Variables <auth/authorization/roles-variables>`_
+   * - `Subscriptions <subscriptions/index>`_
+     - 
+     - `Permission Rules <auth/authorization/permission-rules>`_
+
+.. list-table::
+   :header-rows: 1
+   :class: toc-table
+
+   * - Migrate & Deploy
+     - Reference
+     - Learn
+   * - `Basics <migrations/basics>`_
+     - `Hasura CLI <hasura-cli/index>`_
```

Comment:
Hasura CLI is more like its own thing rather than a reference. It's usually used in the context of migrations & metadata. Maybe it can go under `Migrate & Deploy`?

Reply:
Thinking about this now. This would make the layout a bit lopsided. Maybe we keep it under reference? It actually is a kind of reference, with the list of commands and info on how they're used.

Reply:
Also easier to find there at the top